components/Formsy/PhoneInput.jsx

```diff
@@ -87,6 +87,22 @@ class PhoneInput extends Component {
     }
   }
 
+  /**
+   * checkPhone
+   *
+   * @param {String}        phone
+   * @returns {Boolean}
+   */
+  checkPhone (phone)  {
```

Comment:
Instead of implementing a custom method `checkPhone` implement a standard formsy method `[validate](https://github.com/christianalfoni/formsy-react/blob/master/API.md#validate) which would be check phone number as per regexp